src/embed.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
```

Comment:
I think rather than encouraging the user to include an external script, we should add an inline `style` attribute to the `iframe` element. This means the user won't potentially try to load the script multiple times if there are multiple embedded diagrams and will be more efficient.